.github/workflows/sil-kit-ci.yml

```diff
@@ -40,38 +40,68 @@ jobs:
         run: |
           python3 ./SilKit/ci/check_files_changed.py ${{ github.repository }} ${{ github.event.number }}
 
+  check-formatting:
+    name: Format checks for SIL Kit sources
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Install clang-format 18
+        id: install-clang18
+        run: |
+          wget https://apt.llvm.org/llvm.sh -P /tmp/
+          chmod +x /tmp/llvm.sh
+          sudo bash -c '/tmp/llvm.sh'
+          sudo apt update
+          sudo apt install -y clang-format-18
+      - uses: actions/checkout@v4
+        with:
+          submodules: false
+      - name: Check Formatting
+        id: formatting-check
+        run: |
+          python3 ./SilKit/ci/check_formatting.py
+        shell: bash
+      - name: Upload Log
+        if: always()
+        id: format-result-upload
+        uses: actions/upload-artifact@v4
+        with:
+          name: FormatLog
+          path: FormatLog.txt
```

Comment:
Is this file created in the `check_formatting.py` script?

Reply:
Just seen the next line which ignores failures - can/should this step be removed?

Reply:
Yeah, this PR is so old, it is probably an artifact of an earlier version. Removed for now 👍